source/_docs/terminus/updates.md

```diff
@@ -28,6 +28,18 @@ mkdir $HOME/terminus
 cd $HOME/terminus
 curl -O https://raw.githubusercontent.com/pantheon-systems/terminus-installer/master/builds/installer.phar && php installer.phar install</code></pre></figure>
   </div>
+
+## Version Life Cycle
```

Comment:
Terminus EOL seems like it should be in its own PR

Reply:
Yea, looks like I forgot to switch to master first. Resolving...